umh-core/pkg/communicator/actions/actions.go

```diff
@@ -129,6 +129,26 @@ func HandleActionMessage(instanceUUID uuid.UUID, payload models.ActionMessagePay
 			actionLogger:          log,
 			systemSnapshotManager: systemSnapshotManager,
 		}
+
+	case models.GetConfigFile:
```

Comment:
(style) Any special reasons why we differ here from other Handlers by calling a constructor instead of the struct Assignment directly ?